ert_shared/storage/database_schema.py

```diff
@@ -49,6 +49,14 @@ class Ensemble(Entity):
     parameter_definitions = relationship(
         "ParameterDefinition", back_populates="ensemble"
     )
+    responses = relationship(
+        "Response",
+        secondary="join(ResponseDefinition, Response, ResponseDefinition.id == Response.response_definition_id)",
+        # primaryjoin="Ensemble.id == Response.id",
```

Comment:
should be removed / add additonal information why it should still be there to document something?

Reply:
Removed. Was trying to define a join where there's an intermediary table, but I wasn't successful. I figured I'd rather remove the need for `ResponseDefinition` in a later PR.